javadoc/pom.xml

```diff
@@ -64,6 +64,18 @@
               </target>
             </configuration>
           </execution>
+          <execution>
+            <id>delete-sources-dir</id>
+            <phase>verify</phase>
+            <goals>
+              <goal>run</goal>
+            </goals>
+            <configuration>
+              <target>
+                <delete dir="${sources-directory}" />
```

Comment:
This kind of fast-delete will cause problems on windows based builds.

Use this ..
``` xml
<delete dir="${sources-directory}" failonerror="false" />
```

as not being able to delete the directory isn't a fatal build issue.